trulens_eval/trulens_eval/schema.py

```diff
@@ -400,6 +404,7 @@ def __init__(
             app_id = obj_id_of_obj(obj=self.dict(), prefix="app")
 
         self.app_id = app_id
+        self.tags = tags
```

Comment:
like here, tags could be confused as local scope or global scope. So have the outer tags renamed to Tags